air/src/proof/mod.rs

```diff
@@ -241,7 +261,7 @@ fn get_proven_security(
     base_field_bits: u32,
     trace_domain_size: usize,
     collision_resistance: u32,
-) -> u32 {
+) -> ProvenSecurityBits {
     let m_min: usize = 3;
```

Comment:
Maybe add a comment re the strategy used to compute m?

Reply:
Added